src/UserSettingsPage.cpp

```diff
@@ -340,26 +344,64 @@ UserSettingsPage::UserSettingsPage(QSharedPointer<UserSettings> settings, QWidge
-        boxWrap(tr("Start in tray"), startInTrayToggle_);
+        boxWrap(
+          tr("Minimize to tray"),
+          trayToggle_,
+          tr("Keep the application running in the background after closing the client window."));
+        boxWrap(tr("Start in tray"),
+                startInTrayToggle_,
+                tr("Start the application in the background without showing the client window."));
         formLayout_->addRow(new HorizontalLine{this});
-        boxWrap(tr("Circular Avatars"), avatarCircles_);
-        boxWrap(tr("Group's sidebar"), groupViewToggle_);
-        boxWrap(tr("Decrypt messages in sidebar"), decryptSidebar_);
-        boxWrap(tr("Show buttons in timeline"), timelineButtonsToggle_);
-        boxWrap(tr("Typing notifications"), typingNotifications_);
-        boxWrap(tr("Sort rooms by unreads"), sortByImportance_);
+        boxWrap(tr("Circular Avatars"),
+                avatarCircles_,
+                tr("Change the appearance of user avatars in chats.\nOFF - square, ON - Circle."));
+        boxWrap(tr("Group's sidebar"),
+                groupViewToggle_,
+                tr("Show a column containing groups and tags next to the room list."));
+        boxWrap(tr("Decrypt messages in sidebar"),
+                decryptSidebar_,
+                tr("Decrypt the messages shown in the sidebar.\nOnly affects messages in "
+                   "encrypted chats."));
+        boxWrap(tr("Show buttons in timeline"),
+                timelineButtonsToggle_,
+                tr("Show buttons to quickly reply, react or access additional options next to each "
+                   "message."));
+        boxWrap(tr("Typing notifications"),
+                typingNotifications_,
+                tr("Show who is typing in a room.\nThis will also enable or disable sending typing "
+                   "notifications to others."));
+        boxWrap(
+          tr("Sort rooms by unreads"),
+          sortByImportance_,
+          tr(
+            "Display rooms with new messages first.\nIf this is off, the list of rooms will only "
+            "be sorted by the timestamp of the last message in a room.\nIf this is on, rooms which "
+            "have active notifications (the small circle with a number in it) will be sorted on "
+            "top. Rooms, that you have muted, will still be sorted by timestamp, since you don't "
+            "seem to consider them as important as the other rooms."));
         formLayout_->addRow(new HorizontalLine{this});
-        boxWrap(tr("Read receipts"), readReceipts_);
-        boxWrap(tr("Send messages as Markdown"), markdownEnabled_);
-        boxWrap(tr("Desktop notifications"), desktopNotifications_);
-        boxWrap(tr("Highlight message on hover"), messageHoverHighlight_);
-        boxWrap(tr("Large Emoji in timeline"), enlargeEmojiOnlyMessages_);
+        boxWrap(tr("Read receipts"), readReceipts_, tr("Show the status of the sent messages."));
```

Comment:
```suggestion
        boxWrap(tr("Read receipts"), readReceipts_, tr("Show if your message was read."));
```

No sure about this one, maybe you have a better idea? I don't like the plain "status", since it is simply not really clear, what is meant by that.